app/services/idv/proofing_components.rb

```diff
@@ -33,7 +33,7 @@ def document_type
     end
 
     def source_check
-      Idp::Constants::Vendors::AAMVA if idv_session.verify_info_step_complete?
+      idv_session.source_check_vendor
```

Comment:
I think we need to tweak to account for the 50/50 state:
```suggestion
      idv_session.source_check_vendor || (Idp::Constants::Vendors::AAMVA if idv_session.verify_info_step_complete?)
```
(you can probably come up with a better / more ruby-ish way to do ☝️ )

Basically, if the key is not in the session, we need to fall back to the old (bad) way

Reply:
Fixed!